src/TrackGenerator.cpp

```diff
@@ -6,19 +6,19 @@
  * @param num_azim number of azimuthal angles in \f$ [0, 2\pi] \f$
  * @param spacing track spacing (cm)
  */
-TrackGenerator::TrackGenerator(Geometry* geometry, const int num_azim,
-                               const double spacing) {
+TrackGenerator::TrackGenerator(Geometry* geometry, int num_azim,
+                               double spacing) {
 
   setNumThreads(1);
-
   _geometry = geometry;
   setNumAzim(num_azim);
-  setTrackSpacing(spacing);
+  setDesiredAzimSpacing(spacing);
```

Comment:
As a side note, from a terminology standpoint, are using "azimuthal spacing" in place of "track spacing" now? If so, update the `Options` class to reflect this change.